be/src/olap/tablet.cpp

```diff
@@ -1531,4 +1532,24 @@ std::shared_ptr<MemTracker>& Tablet::get_compaction_mem_tracker(CompactionType c
     }
 }
 
+Status Tablet::get_dict_data(std::set<std::string>& dict_words, int col_id) {
+    std::vector<BetaRowset*> beta_row_sets;
+    {
+        std::shared_lock rdlock(get_header_lock());
+        for (auto entry : _rs_version_map) {
```

Comment:
std::shared_lock rdlock(get_header_lock());
        for (auto entry : _rs_version_map) {
       RETURN_IF_NOT_OK( rs->get_dict_data(dict_words, col_id));
            }